pysgrid/sgrid.py

```diff
@@ -252,6 +268,297 @@ def _save_common_components(self, nc_file):
                     dataset_grid_var.axes = ' '.join(axes)
         return grid_vars
 
+    def get_grid(self, grid='node'):
```

Comment:
I have some overarching concerns about this. As @hrajagers spoke to in issue #76, the SGRID conventions where drafted to:
> The sgrid conventions have been drafted to be
> able to get rid of the multiple separate grid definitions for centre,
> corner, u, v "points"

From reading through the thread of that issue, it doesn't seem the discussion regarding whether there's a single grid, or separate grids was resolved in #76 (or in another issue as suggested by @ocefpaf).